keycloak/keycloak-admin-ui
# Realm settings(themes): Add themes tab + tests

## Motivation

https://issues.redhat.com/browse/APPDUX-939

## Brief Description

Adds themes tab, cypress tests for Realm Settings

## Verification Steps


1. Go to Realm Settings -> Themes tab.
2. Populate fields and save.
3. Navigate to another location in Keycloak and return to Realm Settings -> Themes tab.
4. Verify that the data saves correctly.

## Checklist:
